scripts/create_postgresDB.sh

```diff
@@ -0,0 +1,4 @@
+echo -e "# # # # # # # START : Creating Postgres Database # # # # # ## #"
+psql postgres -tc "SELECT 1 FROM pg_database WHERE datname = 'superset'" | grep -q 1 || psql postgres -c "CREATE DATABASE superset"
+echo -e "# # # # # # # START : Creating PostGres User# # # # # # #"
+psql postgres -tAc "SELECT 1 FROM pg_roles WHERE rolname='postgresuser'" | grep -q 1 || psql postgres -c "CREATE USER postgresuser WITH PASSWORD 'pguserpassword';"
```

Comment:
do  we don't need to drop db if already exist ,so that on each unit test fresh db created like-

DROP DATABASE IF EXISTS superset;

Reply:
I have added step for dropping the database